src/main/java/controller/UserController.java

```diff
@@ -0,0 +1,108 @@
+package controller;
+
+import model.UserModel;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.ModelAndView;
+import pack.PostgresDriverManager;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+@Controller
+public class UserController {
+    @RequestMapping
+    public String start() {
+        return "/start";
+    }
+
+    @PostMapping(path = "/create", consumes = {MediaType.APPLICATION_FORM_URLENCODED_VALUE})
+    public ModelAndView post(UserModel model) {
```

Comment:
Why all methods are named `post`?)

Reply:
I use post methods to make changes in the database
if I need to take something,  I use get

Reply:
I mean naming of method
we should call it something like `createUser` etc.